src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -201,7 +202,7 @@ public ProcessResult updateCase(
                 exception
             );
         // exceptions received from case update client
-        } catch (RestClientException exception) {
+        } catch (RestClientException | ConstraintViolationException exception) {
```

Comment:
No need, I think.
As long as the log level is 'error'  we're good.

Reply:
This could be helpful to be more specific. This is particularly important here, where behind one error message there are two kinds of problems (that should be solved):
- failure to transform exception record details into updated case details
- failure to call ccd

Reply:
this section would only be for update call. ccd calls will result in feign exceptions only

Reply:
Yes, but it's the same message as for other sections.

Reply:
If we don't make custom message for this kind of situation, better not to catch this exception explicitly, as it doesn't matter. Actually, the presence of this whole catch bit is unnecessary and duplicates the handling of `Exception`.

Reply:
To clarify, `caseUpdateClient` is not talking to CCD, it's talking to case update endpoints in services using bulk-scan

Reply:
handling `ConstraintViolationException` separately now